mentii/mentii
# (Teacher) Remove student from class

Allows a teacher to remove a student from a class. Sends a notification to the student they have been removed from the class.

**Significant Changes**
1. Added ability to remove student from within a class
2. An email is sent to a student once removed from a class
3. Slight refactoring in **db_utils** to handle removal of attributes
4. Updated unit test for update_item. Unit tests for remove student

**How to Test**
1. Login as teacher or admin
2. Create a class
3. Login with an account that has your email and join the newly created class
4. Log back in as teacher or admin and see that user is in class list with **X Remove Student**
5. Click link and remove student
6. Check you got an email saying you were removed from the class
7. Refresh class details page and ensure student is not there anymore
8. Login as student and check that you don't have the class in your dashboard and can rejoin the class

**Notes**
:taco:
